terraform/deployments/govuk-publishing-infrastructure/csvs_s3.tf

```
@@ -0,0 +1,35 @@
+resource "aws_iam_policy" "csvs_bucket_polices" {
+  name        = "csvs_s3"
+  description = "Read and write to this environment's CSVS s3 buckets"
+
+  policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Effect = "Allow"
+        Action = [
+          "s3:GetBucketLocation",
+          "s3:ListBucket",
+        ]
+        Resource = ["arn:aws:s3:::${var.govuk_environment}content-data-csvs", "arn:aws:s3:::${var.govuk_environment}-specialist-publisher-csvs", "arn:aws:s3:::${var.govuk_environment}-support-api-csvs", "arn:aws:s3:::${var.govuk_environment}-whitehall-csvs"]
```

Slightly more readable like this?

```suggestion
        Resource = [
          "arn:aws:s3:::${var.govuk_environment}content-data-csvs",
          "arn:aws:s3:::${var.govuk_environment}-specialist-publisher-csvs",
          "arn:aws:s3:::${var.govuk_environment}-support-api-csvs",
          "arn:aws:s3:::${var.govuk_environment}-whitehall-csvs",
        ]
```

(and similarly below)

---

Yeah good shout, will get them updated